addons/stock/models/stock_orderpoint.py

```diff
@@ -500,11 +500,11 @@ def _unlink_processed_orderpoints(self):
         domain = [
             ('create_uid', '=', SUPERUSER_ID),
             ('trigger', '=', 'manual'),
-            ('qty_to_order', '<=', 0)
```

Comment:
`qty_to_order` is `stored=True` in v16 and v17. I guess this change only makes sense in v18+. I also assume your performance numbers is from that version as well?